### PR TITLE
stdydx + sttia

### PR DESCRIPTION
**Pull request by riley-stride**
- add stdydx, sttia
- add fee currencies so users can pay gas in stdydx, dydx, tia, sttia

### CODE REVIEW

**Comment by HeesungB** [comment]
@riley-stride Can you delete the coingeckoid field that can't get a value?

File: cosmos/stride.json [hunk 1]
```diff
@@ -80,6 +80,20 @@
       "coinGeckoId": "stride-staked-injective",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/stinj.png"
     },
+    {
+      "coinDenom": "stDYDX",
+      "coinMinimalDenom": "stadydx",
+      "coinDecimals": 18,
+      "coinGeckoId": "stride-staked-dydx",
```
Comment (HeesungB) [inline cosmos/stride.json:87]: @riley-stride Could you remove `coinGeckoId` field?

File: cosmos/stride.json [hunk 2]
```diff
@@ -80,6 +80,20 @@
...
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/stdydx.png"
+    },
+    {
+      "coinDenom": "stTIA",
+      "coinMinimalDenom": "stutia",
+      "coinDecimals": 6,
+      "coinGeckoId": "stride-staked-tia",
```
Comment (HeesungB) [inline cosmos/stride.json:94]: @riley-stride Could you remove coinGeckoId field?

File: cosmos/stride.json [hunk 3]
```diff
@@ -99,63 +113,152 @@
...
+          "average": 0.001,
+          "high": 0.01
+        }
+      },
+      {
+        "coinDenom": "OSMO",
+        "coinMinimalDenom": "ibc/D24B4564BCD51D3D02D9987D92571EAC5915676A9BD6D9B0C1D0254CB8A5EA34",
+        "coinDecimals": 6,
+        "coinGeckoId": "osmosis",
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/osmosis/uosmo.png",
```
Comment (HeesungB) [inline cosmos/stride.json:145]: @riley-stride  You can only add coinImageUrls that are in the identifier of that chain.

Please add `uosmo.png` to the `images/stride` directory and modify the path.

Comment (riley-stride) [reply cosmos/stride.json:143]: done

File: cosmos/stride.json (hunk 2 above)
Comment (riley-stride) [reply cosmos/stride.json]: done

File: cosmos/stride.json [hunk 4]
```diff
@@ -99,63 +113,152 @@
...
+          "low": 20000000000,
+          "average": 20000000000,
+          "high": 20000000000
+        }
+      },
+      {
+        "coinDenom": "stEVMOS",
+        "coinMinimalDenom": "staevmos",
+        "coinDecimals": 18,
+        "coinGeckoId": "stride-staked-evmos",
```
Comment (HeesungB) [inline cosmos/stride.json:209]: @riley-stride Could you remove coinGeckoId field?

**Comment by riley-stride** [comment]
> @riley-stride Can you delete the coingeckoid field that can't get a value?

Of course. I've just removed it.

File: cosmos/stride.json [hunk 5]
```diff
@@ -99,63 +113,152 @@
...
+          "low": 0.01,
+          "average": 0.01,
+          "high": 0.01
+        }
+      },
+      {
+        "coinDenom": "stTIA",
+        "coinMinimalDenom": "stutia",
+        "coinDecimals": 6,
+        "coinGeckoId": "",
```
Comment (HeesungB) [inline cosmos/stride.json:231]: @riley-stride Could you remove empty `coinGeckoId` field?

**Comment by HeesungB** [comment]
@riley-stride And Image files should be resized 256x256px

File: cosmos/stride.json [hunk 6]
```diff
@@ -99,63 +111,152 @@
...
+          "low": 15000000000,
+          "average": 15000000000,
+          "high": 15000000000
+        }
+      },
+      {
+        "coinDenom": "stDYDX",
+        "coinMinimalDenom": "stadydx",
+        "coinDecimals": 18,
+        "coinGeckoId": "stride-staked-dydx",
```
Comment (HeesungB) [inline cosmos/stride.json:253]: @riley-stride Can you please delete the coinGeckoId?
This should be added when you can see the price.

Comment (riley-stride) [reply cosmos/stride.json]: makes sense. done.

**Comment by riley-stride** [comment]
> @riley-stride And Image files should be resized 256x256px

Done. 256h PNGs.

File: cosmos/stride.json (hunk 5 above)
Comment (riley-stride) [reply cosmos/stride.json]: done

File: cosmos/stride.json (hunk 4 above)
Comment (riley-stride) [reply cosmos/stride.json]: done

**Pull request merged**